utils/files.go

```diff
@@ -3,10 +3,17 @@ package utils
 import (
 	"os"
 	"path/filepath"
+	"runtime"
 
 	"github.com/google/uuid"
 )
 
-func TempFileName(prefix, suffix string) string {
-	return filepath.Join(os.TempDir(), prefix+uuid.NewString()+suffix)
+func RandomSocketOrFileName(prefix, suffix string) string {
```

Comment:
I rather leave this function (`TempFileName`) alone and create a `SocketName` in move this function into the `core/playback/mpv` package, also selectable by build tags

Reply:
sounds good